crates/sui-protocol-constants/src/lib.rs

```diff
@@ -105,9 +105,12 @@ pub const OBJ_METADATA_COST_NON_REFUNDABLE: u64 = 50;
 
 /// === Tokenomics ===
 
-// TODO: this should be changed to u64.
 /// Sender of a txn that touches an object will get 99% of the storage rebate back.
-pub const STORAGE_REBATE_RATE: f64 = 0.99;
+///
+/// STORAGE_REBATE_RATE is a pair of integers, representing a fraction of a percentage
+/// value. Use for deterministic computation with a multiplication of the first value
+/// divided by the second. Values can have any number of digits and precision.
+pub const STORAGE_REBATE_RATE: (u64, u64) = (99, 100);
```

Comment:
ridiculously verbose comment....

Reply:
Other rates in this file are in basis point. Do we want to be consistent here too?

Reply:
I do not know and good point. It was not that, but that does not mean it should not be.
To me it's unclear what that value wants to be and what are the possible variation.

Reply:
I am actually liking the idea and I will move the code to do that.
After all when we are changing those values we are updating the code for now, so it does not really matter....